### DISCUSSION TITLE
Capture data on previous responses to present them later.

### DISCUSSION THREAD

**Discussion by danielpetinga** [ditdot-dev/vue-flow-form#286] [General] [answered]
Hello guys, everything good? I have a question about how to take the name from the previous answer and insert it into the tagline of the next question. Could anyone give me some help? Here's the code:

new QuestionModel({
	id: 'nome_masculino',
	tagline: 'Nós gostaríamos de conhecê-lo.',
	title: 'Qual é o seu nome?',
	type: QuestionType.Text,
	required: true,
	placeholder: 'Por favor, digite o seu nome aqui...',			
}),
new QuestionModel({
	id: 'email',
	tagline: "Nice to meet you 😊, let's continue",
	title: 'Fornecer um exemplo de e-mail.',
	type: QuestionType.Email,
	required: true,
	placeholder: 'Comece a digitar aqui...'
}),

**Comment by spinn** [comment, marked as answer]
You can accomplish this by defining questions as components, documentation here:

https://www.ditdot.hr/en/docs/vue-flow-form-guide#question-components
